docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.text.scss

```diff
@@ -33,12 +45,36 @@
       font-size: hb-calculate-rems(38px);
     }
   }
+
+  @include hb-traditional {
+    font-family: $hb-traditional-font--serif;
+    font-weight: hb-theme-font-weight(regular);
+    font-size: hb-calculate-rems(48px);
+    line-height: 112%;
+
+    @include grid-media-min('md') {
+      font-size: hb-calculate-rems(33px);
+      line-height: 125%;
+    }
+  }
 }
 
 @mixin hb-heading-3 {
   @include hb-colorful {
     @include hb-block-heading;
```

Comment:
I wish I understood this block-heading more, so I knew if we should be making traditional changes in there too...

Reply:
it's used hardly anywhere. I have a feeling it was an idea that maybe wasn't as applicable as we hoped

Reply:
This might be good to bring up in a dev-huddle. I could see us documenting the use-case in a comment better, or even removing it, if that makes more sense.

Reply:
Either way, no need to take action on it for this card. I'll make a note to bring it up in the future.